xdsl/dialects/irdl/pyrdl_to_irdl.py

```diff
@@ -69,7 +69,7 @@ def attr_def_to_irdl(
         param_values.append(constraint_to_irdl(builder, param[1]))
     builder.insert(ParametersOp(param_values))
 
-    return AttributeOp(attr_def.name, Region([block]))
+    return AttributeOp(attr_def.name.split(".", 1)[1], Region([block]))
```

Comment:
Can you please add a helper function with a test somewhere? Maybe on `Dialect` itself, as a static method? Something like `Dialect.split_name(name: str) -> tuple[str, str]`